examples/example_spider_walking/main.cpp

```diff
@@ -39,7 +39,7 @@ using gtsam::noiseModel::Unit;
 using namespace gtdynamics;
 
 // Returns a Trajectory object for a single spider walk cycle.
-Trajectory getTrajectory(vector<string> links, Robot robot, size_t repeat) {
+Trajectory getTrajectory(vector<string>& links, size_t repeat) {
```

Comment:
Pass in `robot` instead of `links`.

Reply:
This comment is here just as a bookmark for the next PR. Same deal for below comments regarding passing in the `robot` directly.